.github/workflows/main.yml

```diff
@@ -6,7 +6,7 @@ jobs:
     name: Regula Terraform
     steps:
     - uses: actions/checkout@master
-    - uses: fugue/regula-action@v1.4.0
+    - uses: fugue/regula-action@release/v1.5.0
```

Comment:
Oh man, I always use the commit hash here until the release is tagged. This is a much better idea!